templates/publisher/_repo-disconnect-modal.html

```diff
@@ -0,0 +1,21 @@
+<div class="p-modal" data-js="repo-disconnect-modal" id="repo-disconnect-modal" style="display: none;">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header">
+      <h4><i class="p-icon--warning"></i>&nbsp;&nbsp;Disconnecting your repository</h4>
+      <button class="p-modal__close" aria-label="Close active modal" aria-controls="repo-disconnect-modal">Close</button>
+    </header>
+    <p>By disconnecting this repository from your snap, the following will occur:</p>
+    <ul>
+      <li>The previous builds you did on the snap may disappear from the Builds page</li>
+      <li>You will be able to continue to release your previous revisions on the Releases tab</li>
+    </ul>
+    <p>Do you wish to <strong>continue disconnecting your repository?</strong></p>
+    <form class="p-form" method="POST" action="/{{ snap_name }}/builds/disconnect">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      <p class="u-align--right">
+        <button aria-label="Close active modal" aria-controls="repo-disconnect-modal"class="p-button--neutral">Cancel</button>
+        <button type="submit" class="p-button--positive">Confirm</button>
```

Comment:
Can we do an immediate disable with a delayed spinner on this button - for me I clicked it and it felt like nothing was happening for a few seconds and I was tempted to click it again.